src/common/dao/project.go

```diff
@@ -156,18 +156,18 @@ func GetProjects(query *models.ProjectQueryParam) ([]*models.Project, error) {
 
 // GetGroupProjects - Get user's all projects, including user is the user member of this project
 // and the user is in the group which is a group member of this project.
-func GetGroupProjects(groupDNCondition string, query *models.ProjectQueryParam) ([]*models.Project, error) {
+func GetGroupProjects(groupIDCondition string, query *models.ProjectQueryParam) ([]*models.Project, error) {
```

Comment:
If it only supports would it be better to input a slice of groupID and transform it into `ID Contidioin` in the func in `dao` package?

Reply:
Updated